server/sample.env

```diff
@@ -1,3 +1,3 @@
 JWT_SECRET=
 MONGO_URI=
-TEAM_NAMES=Bonnie,Sina
\ No newline at end of file
+TEAM_NAMES=Bonnie,Sina,Sebastien
```

Comment:
This code is from the previous PR. Please adjust the base of the PR or rebase to only reflect changes pertaining to this PR